docs/table/access_table.rst

```diff
@@ -288,6 +288,19 @@ selected rows or columns.  ::
        9.000  11
       12.000  14
 
+We can select rows from a table using conditionals to create boolean masks. A
+table indexed with a boolean array will only return rows where the mask array
+is equal to `True`. Different conditionals can be combined using the bitwise
+operators.  ::
+
+  >>> print(t[(t['a'] > 4) & (t['b'] > 8)])  # Table rows where column a > 4
```

Comment:
This expression is a little dense.  For clarity I would suggest writing this as:
```
>>> mask = (t['a'] > 4) & (t['b'] > 8)  # Table rows where column a > 4 and b > 8
>>> print(t[mask])
```
I actually do this separation most of the time in my production code.